src/lib/mina_base/account_timing.ml

```diff
@@ -118,39 +118,39 @@ let to_bits t =
         } =
     to_record t
   in
-  is_timed
-  :: ( Balance.to_bits initial_minimum_balance
-     @ Global_slot.to_bits cliff_time
-     @ Amount.to_bits cliff_amount
-     @ Global_slot.to_bits vesting_period
-     @ Amount.to_bits vesting_increment )
+  let open Random_oracle_input.Chunked in
+  Array.reduce_exn ~f:append
+    [| packed ((if is_timed then Field.one else Field.zero), 1)
```

Comment:
I get the feeling this is used to switch something on or off depending if the account is timed.  i.e. present or not present depending on that.